components/services/brave_wallet/zcash/BUILD.gn

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//mojo/public/tools/bindings/mojom.gni")
+
+static_library("zcash_decoder") {
```

Comment:
component instead? I think we should have visibility setting for this target too, it should only be accessed by the service lib.